.ci/windows/index.html

```diff
@@ -0,0 +1 @@
+<p>Over the past 7 days, these are the top 10 failing tests:</p><table><tr><td colspan=2>test_restart_recursive_glob</td><td>failed 4 times:</td></tr><tr><td> </td><td><a href=https://beats-ci.elastic.co/job/Beats/job/beats/job/7.10/141/console rel=nofollow>elastic&#43;beats</a></td><td>failed 4 times</td></tr><tr><td colspan=2>TestFetch</td><td>failed 3 times:</td></tr><tr><td> </td><td><a href=https://beats-ci.elastic.co/job/Beats/job/beats/job/7.10/131/console rel=nofollow>elastic&#43;beats</a></td><td>failed 3 times</td></tr><tr><td colspan=2>TestFleetGateway</td><td>failed 2 times:</td></tr><tr><td> </td><td><a href=https://beats-ci.elastic.co/job/Beats/job/beats/job/7.9/260/console rel=nofollow>elastic&#43;beats</a></td><td>failed 2 times</td></tr><tr><td colspan=2>TestFleetGateway/Periodically_communicates_with_Fleet</td><td>failed 2 times:</td></tr><tr><td> </td><td><a href=https://beats-ci.elastic.co/job/Beats/job/beats/job/7.9/260/console rel=nofollow>elastic&#43;beats</a></td><td>failed 2 times</td></tr><tr><td colspan=2>TestFleetGateway/Successfully_connects_and_receives_a_series_of_actions</td><td>failed 2 times:</td></tr><tr><td> </td><td><a href=https://beats-ci.elastic.co/job/Beats/job/beats/job/7.9/260/console rel=nofollow>elastic&#43;beats</a></td><td>failed 2 times</td></tr><tr><td colspan=2>TestFleetGateway/send_no_event_and_receive_no_action</td><td>failed 2 times:</td></tr><tr><td> </td><td><a href=https://beats-ci.elastic.co/job/Beats/job/beats/job/7.9/260/console rel=nofollow>elastic&#43;beats</a></td><td>failed 2 times</td></tr><tr><td colspan=2>TestIncludedExcludedFiles</td><td>failed 2 times:</td></tr><tr><td> </td><td><a href=https://beats-ci.elastic.co/job/Beats/job/beats/job/master/706/console rel=nofollow>elastic&#43;beats</a></td><td>failed 2 times</td></tr><tr><td colspan=2>test_base</td><td>failed 2 times:</td></tr><tr><td> </td><td><a href=https://beats-ci.elastic.co/job/Beats/job/beats/job/master/706/console rel=nofollow>elastic&#43;beats</a></td><td>failed 2 times</td></tr><tr><td colspan=2>test_close_renamed</td><td>failed 2 times:</td></tr><tr><td> </td><td><a href=https://beats-ci.elastic.co/job/Beats/job/beats/job/7.x/594/console rel=nofollow>elastic&#43;beats</a></td><td>failed 2 times</td></tr><tr><td colspan=2>ExampleWrapper</td><td>failed 1 times:</td></tr><tr><td> </td><td><a href=https://beats-ci.elastic.co/job/Beats/job/beats/job/7.x/613/console rel=nofollow>elastic&#43;beats</a></td><td>failed 1 times</td></tr></table>
```

Comment:
Can we remove this? Seems like a leftover.

Reply:
Yep - looks like it crept in https://github.com/elastic/apm-agent-dotnet/commit/85295009ccadf73b8501200a2810526ace58f26d

Reply:
I removed it.